Dockerfile

```diff
@@ -17,29 +17,22 @@ ENV IMAGETAG=$tag
 RUN echo " ---------------------------------- "
 RUN echo "env variable IMAGETAG is ${IMAGETAG}"
 
+# create default scientific Python environment
 
-
-
-    # create default scientific Python environment
-
-RUN conda config --add channels pyviz/label/dev
-RUN conda config --add channels bokeh/label/dev
-RUN conda config --add channels intake
 RUN conda config --add channels bioconda
 RUN conda config --add channels conda-forge
-#RUN conda update --yes conda
+RUN conda config --set channel_priority strict
+RUN conda update --yes conda
 
-RUN conda env update  --file /tmp/environment.yml --prune
+RUN conda env update --file /tmp/environment.yml
```

Comment:
We are installing this into the base environment, is that right?

Reply:
I.e., I think the environment name from within the environment file will be ignored.